pkg/network/state.go

```diff
@@ -913,6 +918,47 @@ func (ns *networkState) determineConnectionIntraHost(connections []ConnectionSta
 	}
 }
 
+// fixConnectionDirection fixes connection direction
+// for UDP incoming connections.
+//
+// Some UDP connections can be assigned an incoming
+// direction incorrectly since we cannot reliably
+// distinguish between a server and client for UDP
+// in eBPF. Both clients and servers can call
+// the system call bind() for source ports, but
+// UDP servers don't call listen() or accept()
+// like TCP.
```

Comment:
I thought we had found that this *wasn't* the. case?

Reply:
This is the case. See the thread in slack.